tests/components/mqtt/test_binary_sensor.py

```diff
@@ -24,6 +26,78 @@
 )
 
 
+logging.basicConfig(level=logging.DEBUG)
+LOG = logging.getLogger()
+
+
+async def test_not_specifying_expire_at_setting(hass, mqtt_mock):
```

Comment:
Instead of duplicating all the code from `test_setting_sensor_value_expires`, can you refactor to a helper function which you call twice, once with "expire_after" set and one where its not?

Reply:
I have removed this code.  The other tests would fail if they couldn't init without the expiry setting.